rules/config

```diff
@@ -161,7 +161,13 @@ ENABLE_AUTO_TECH_SUPPORT = y
 # ENABLE_TRANSLIB_WRITE = y
 
 # INCLUDE_MACSEC - build docker-macsec for macsec support
-INCLUDE_MACSEC = y
+INCLUDE_MACSEC ?= y
```

Comment:
Do we want to change macsec too?

Reply:
thanks, good catch! I updated INCLUDE_MACSEC instead of INCLUDE_ROUTER_ADVERTISER